src/library/dao/trackdao.cpp

```diff
@@ -1870,10 +1914,10 @@ void TrackDAO::detectCoverArtForTracksWithoutCover(volatile const bool* pCancel,
     QSqlQuery updateQuery(m_database);
     updateQuery.prepare(
         "UPDATE library SET "
-        "  coverart_type=:coverart_type,"
-        "  coverart_source=:coverart_source,"
-        "  coverart_hash=:coverart_hash,"
-        "  coverart_location=:coverart_location "
+        "coverart_type=:coverart_type,"
```

Comment:
Is a tailing " " lost here?

Reply:
Every line ends with either ' ' or with ',' consistently.

Reply:
Oh, now I see. Thanks